### ISSUE TITLE
[Discontinued] Cannot deploy new versions

### ISSUE THREAD

**Issue by jorgebodega** [w3tecch/typeorm-seeding#201]
Hi!

As some of you may know, I've been working on this library for the last months, trying to solve issues and refactoring, because I use this library every day at my job to seed tests. I've encountered so many problems, and some of the pieces of the code were so old, and I tried to do my best.

But finally there was a problem: @hirsch88 is unavailable and we need the original author to generate a NPM token to deploy new versions, like `1.6.2` or `next` (probably `2.0.0`), so after a lot of efforts, I've decided to release new versions under a new banner:

[@jorgebodega/typeorm-seeding](https://www.npmjs.com/package/@jorgebodega/typeorm-seeding)

Could be installed with:

```bash
npm i [-D] @jorgebodega/typeorm-seeding
yarn add [-D] @jorgebodega/typeorm-seeding
```

I will try to visit this repo periodically, for if there are some new issues or ideas, but it would be if any new issue or pull request are created on that new repository.

Please, be kind. This is my first **public** package, and is currently on dev (`2.0.0-next.3`), but with some changes. This week I will prepare new documentation, CHANGELOG and so on.

P.D.: I'm not stealing this library. Whenever someone with permissions over this library returns, I will send a PR if they want.

Cheers!

**Comment by DanielRamosAcosta** [comment]
ping @hirsch88

**Comment by xinghul** [comment]
hello @jorgebodega, just to clarify, are you planning on swapping out `Faker` and replacing it with `@faker-js/faker`?

**Comment by jorgebodega** [comment]
No plans right now, at least here withot deploy permission. Please check #204

**Comment by hirsch88** [comment]
hi @jorgebodega just added you as a npm maintainer hope that helps with the releasing.

**Comment by nicolasrouanne** [comment]
@jorgebodega @hirsch88 where do we go from here? 
Does this becomes the official repository back again? While merging something like 225 commits from https://github.com/jorgebodega/typeorm-seeding into https://github.com/w3tecch/typeorm-seeding

I guess it would probably be the preferred option from the community, since this original project still has 40 times the amount of download of the fork 🙏🏻

**Comment by acSpock** [comment]
@jorgebodega I agree with @nicolasrouanne, any plans to merge the forked repo back into w3tecch now that you're a maintainer? I imagine you could merge it back and cut a major release and go from there. Would like to know going forward as upgrading to the latest typeorm breaks the current version of w3tecch/typeorm-seeding.

**Comment by Jedliu** [comment]
Agree. It's hard to notice this difference until finding this issue. Better to make a decision and put it in the README to avoid confusion. Thank you for your hard work and looking forward to the new version update to accommodate Typeform 0.3.*

**Comment by jorgebodega** [comment]
Sorry @acSpock, please see https://github.com/jorgebodega/typeorm-seeding/issues/129#issuecomment-1101522888

Also @Jedliu, that is a good idea.

**Comment by divmgl** [comment]
What's going on with this package? @jorgebodega and @hirsch88, have you come to an agreement on a path forward with this package? It looks like the last mention of a block in communication was here, back in April: https://github.com/jorgebodega/typeorm-seeding/issues/129#issuecomment-1101522888

@hirsch88 I see that you're active on GitHub. Can we all coordinate to get these packages merged into one? This will just take a few hours of both of your time. I'd be happy to jump on a Discord call to help.

I'm looking into `typeorm-seeding` for my company and I'd really rather not write my own seeder and would much prefer to contribute to an existing one. I'm now needing a seeder for tests, but this package in its current state is basically unusable for seeding data in tests.

**Comment by divmgl** [comment]
@jorgebodega and @hirsch88 I see you're both active on GitHub. Can we coordinate to get https://github.com/jorgebodega/typeorm-seeding merged into this package (or a new plan for this package moving forward)? I'd be happy to jump on a Discord call if needed, thanks.
